nodejs/lancedb/connection.ts

```diff
@@ -65,6 +66,8 @@ export interface CreateTableOptions {
    * The available options are described at https://lancedb.github.io/lancedb/guides/storage/
    */
   storageOptions?: Record<string, string>;
+  schema?: Schema;
+  embeddingFunction?: EmbeddingFunctionConfig;
```

Comment:
Does `open_table` load the embedding config back from the schema?

Reply:
So that's actually a major issue I found while writing some additional tests. When deserializing the table using arrow's functions, it doesn't preserve the metadata like it does in other languages. So I think there's a bit more work that needs to be done on this PR before it can be merged.